lab_1/main.py

```diff
@@ -69,7 +108,41 @@ def get_concordance(tokens: list, word: str, left_context_size: int, right_conte
     right_context_size = 3
     --> [['man', 'is', 'happy', 'the', 'dog', 'is'], ['dog', 'is', 'happy', 'but', 'the', 'cat']]
     """
-    pass
+    conco_all = []
+    index_word = 0
+
+    if checker(tokens, list) and checker(word, str) \
```

Comment:
Usually, there are checks for arguments and body function - separated.
And body function is on the same level as checks.